lua/core/autopairs.lua

```diff
@@ -1,51 +1,60 @@
--- if not package.loaded['nvim-autopairs'] then
---   return
--- end
-local status_ok, _ = pcall(require, "nvim-autopairs")
-if not status_ok then
-  return
+local M = {}
+
+M.config = function()
+  lvim.builtin.autopairs = {
+    active = true,
+  }
 end
-local npairs = require "nvim-autopairs"
-local Rule = require "nvim-autopairs.rule"
 
--- skip it, if you use another global object
-_G.MUtils = {}
+M.setup = function()
+  local status_ok, _ = pcall(require, "nvim-autopairs")
```

Comment:
You can remove this up to L13. Since this is only called once the plugin is loaded, it is always available